src/git.ts

```diff
@@ -51,6 +51,10 @@ export async function gitTag(operation: Operation): Promise<Operation> {
     // See https://git-scm.com/docs/git-tag
     '--annotate',
 
+    // Sign the tag with a GPG key, if one is configured
+    // See https://git-scm.com/docs/git-tag#Documentation/git-tag.txt---sign
+    '--sign', // Sign the tag with a GPG key
```

Comment:
I don't think it should pass by default, as not everybody would like to sign their commit. We could pass through when user does `bumpp --sign` instead.

Reply:
I didn't know that ! Thanks 

I close this PR